src/controllers/expenses.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const expensesService = require('../services/expenses');
+const usersService = require('../services/users');
+
+const getAll = (req, res) => {
+  const expenses = expensesService.getAll(req.query);
+
+  res.send(expenses);
+};
+
+const getOne = (req, res) => {
+  const { expenseId } = req.params;
+
+  const foundExpense = expensesService.getById(+expenseId);
```

Comment:
```suggestion
  const foundExpense = expensesService.getById(Number(expenseId));
```
For better reading and understanding your code